Algorithmization/src/com/company/task_1/Task_8.java

```diff
@@ -0,0 +1,30 @@
+package com.company.task_1;
+
+import java.util.Arrays;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Task_8 {
+    private static int[] removeMin(int[] a, int r) {
+        return Arrays.stream(a).filter(i -> i != r).toArray();
+    }
+
+    private static int min(int[] a) {
+        return Arrays.stream(a).min().orElse(Integer.MIN_VALUE);
```

Comment:
using `Integer.MIN_VALUE` is not correct here. you will receive empty optional only if the provided stream is empty, it should not be empty , but if it will - it's an exceptional case, so you can write it like `Arrays.stream(a).min().orElseThrow(IllegalArgumentException::new);`

Reply:
Исправил